tests/ci/release.py

```diff
@@ -78,7 +77,15 @@ def read_version(self):
         self._git.update()
         self.version = get_version_from_repo()
 
+    def check_prerequisites(self):
+        """
+        Check tooling installed in the system
+        """
+        self.run("gh auth status")
+        self.run("git status")
```

Comment:
Actually, this particular line doesn't make sense since `ClickHouseVersion` heavily depends on `git` command.

Reply:
It also checks that current directory contains initialized git

Reply:
Still unnecessary, see https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/version_helper.py#L39 -> https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/git_helper.py#L63